src/redux/carSlice.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Please try to uncomment this. The error is thrown because you are reassigning the state. In reality, redux state are immutable, so if you want to change state, it's better to use object destructuring. for example, instead of using 
```
state.status = 'loading'
```
you can use:
```
{
...state,
status: 'loading',
}
```